vuepress/docs/next/tutorials/create/ms/add-access-controls.md

```diff
@@ -1,109 +1,161 @@
+The basic security setup for a blueprint-generated application allows any authenticated user to access the functionality contained in the MFEs and/or microservices. This tutorial defines two user roles for our application:
 
-## Tutorial
-Let's start by securing the list of Conferences so only our two user roles can view the list.
+- `conference-user`: Permitted to view the Conferences in the tableWidget
+- `conference-admin`: Permitted to view Conferences in the tableWidget, and also to delete Conferences from the tableWidget 
 
-1. Edit `ConferenceResource.java` located in the `src/main/java/com/<ORG>/<NAME>.web.rest` directory. Modify the REST API `Conference:getAllConferences` method by adding the following annotation. 
+
+## Apply and Verify Access Controls
+
+### Step 1: Secure the Conference list
+
+The list of Conferences must be visible to only the `conference-user` and `conference-admin` user roles. 
+
+1. Go to the `src/main/java/com/YOUR-ORG/YOUR-NAME.web.rest` directory
+2. Open `ConferenceResource.java` 
+3. Modify the REST API `Conference:getAllConferences` method by adding the following annotation:
 ```
     @PreAuthorize("hasAnyAuthority('conference-user','conference-admin')")
     public List<Conference> getAllConferences() {
 ```
-See the [Spring Security documentation](https://spring.io/projects/spring-security) for more details but this restricts use of the `getConference` method to users who have been assigned either the `conference-user` or the `conference-admin` role on the Keycloak client configured for the microservice. In local testing this defaults to the `internal` client but see notes below on how that works in production.
+This confines use of the `getConference` method to users who are assigned either the `conference-user` or the `conference-admin` role on the Keycloak client configured for the microservice. 
 
-Now we should verify this security check is working.
+> Note: In local testing, the default client is `internal`. Refer to the [Spring Security documentation](https://spring.io/projects/spring-security) for more information.
 
-2. Start up your Keycloak, tableWidget MFE, and microservice. See [these instructions](./run-local.md) if you need a refresher but these are the basic commands using the ent CLI and Docker for keycloak.
-```
+### Step 2: Run your project in a local developer environment
+The following commands leverage the [ent CLI](../../../docs/reference/entando-cli.md).
+
+1. Start up your Keycloak instance
+``` sh
 ent prj ext-keycloak start
-ent prj be-test-run
 ```
-Using a separate cmdline:
+2. Initialize the tableWidget MFE
```

Comment:
1. Step #2 is for the microservice since be=backend.  The order is correct, keycloak then backend then frontend, so just flip the labels. 
2. ent prj fe-test-run doesn't initialize the MFE, it starts it.
3. Maybe we should say "Use a new shell" instead of "new cmd line".  "cmd line" is an odd abbreviation...

Reply:
@nshaw an abbreviation that misled me... could not switching shells have contributed to my issues somehow? kind of doubt it, but i followed the instructions and used a new command line... as if there were other options... i just thought you couldn't send all those commands at once for some reason; like you could send the first two together but the third would have to be sent separately

Reply:
@Lyd1aCla1r3 I'm very sure that doesn't account for what we looked at together. The distinction is that fe-test-run needs to be left running in one shell, and be-test-run needs to be left running in another shell. Otherwise you'd have to use Ctrl-C to exit the process and that would shut it down. You could do this all in 2 shells but I personally tend to use 3, one for keycloak, one for MFE, one for MS. Because keycloak keeps running in the background, you can also use that shell to run the MFE - as long as keycloak successfully started. :(